src/views/home/connect.js

```diff
@@ -5,6 +5,7 @@ module.exports = async function({ ticket, connections }) {
         <div id=page>
             <div class=page-title><div><h1>Connect 🤝</h1></div></div>
             <div class=page-body class=narrow>
+            <div class="cta"><a id="copy-sharing-url-button" href="/home/connect?add_connection=${ c.conn_hash }">Copy Sharing URL 🔗➡️📋</a></div>
```

Comment:
This isn't great, but it works I believe. It can be moved wherever works best or changed.